src/Bicep.Core/SemanticModel/SemanticDiagnosticVisitor.cs

```diff
@@ -1,16 +1,16 @@
-// Copyright (c) Microsoft Corporation.
+﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 using System.Collections.Generic;
 using Bicep.Core.Diagnostics;
 using Bicep.Core.TypeSystem;
 
 namespace Bicep.Core.SemanticModel
 {
-    public class SemanticErrorVisitor : SymbolVisitor
+    public class SemanticDiagnosticVisitor : SymbolVisitor
```

Comment:
>SemanticDiagnosticVisitor [](start = 17, length = 25)

After this change, this will only collect errors from error symbols and duplicate identifier errors, right?

Reply:
Yes, although they don't have to be errors